portafly/src/dal/accounts/getDeveloperAccounts.ts

```diff
@@ -0,0 +1,17 @@
+import { useFetch } from 'react-async'
+import { IAccounts } from 'types'
+
+/**
+ * Get all Applications asynchronously
+ * @returns An object containing: { accounts, error, isPending }
+ */
+function useGetDeveloperAccounts() {
+  const { data, error, isPending } = useFetch<[IAccounts]>(
+    '/developer/accounts',
+    { headers: { Accept: 'application/json' } }
```

Comment:
does it accept .yml?

Reply:
I don't think so, the API is supposed to be JSON, or is it?

Reply:
I was thinking in our strings file, not the api, you are right